modules/internal/ChapelTuple.chpl

```diff
@@ -114,10 +142,20 @@ module ChapelTuple {
     return __primitive("get svec member", this, i);
   }
 
+  /*
```

Comment:
I don't think this should be publicly documented. If anything I think you should add a module level comment about the `--[no-]warn-tuple-iteration` flag with this code example. That flag ultimately controls this param, and so I don't think we want users to be aware of it